lib/active_storage_validations/matchers/limit_validator_matcher.rb

```diff
@@ -0,0 +1,139 @@
+# frozen_string_literal: true
+
+require_relative 'concerns/active_storageable.rb'
+require_relative 'concerns/allow_blankable.rb'
+require_relative 'concerns/attachable'
+require_relative 'concerns/contextable.rb'
+require_relative 'concerns/messageable.rb'
+require_relative 'concerns/rspecable.rb'
+require_relative 'concerns/validatable.rb'
+
+module ActiveStorageValidations
+  module Matchers
+    def validate_dimensions_of(name)
+      DimensionValidatorMatcher.new(name)
+    end
+
+    class DimensionValidatorMatcher
+      include ActiveStorageable
+      include AllowBlankable
+      include Attachable
+      include Contextable
+      include Messageable
+      include Rspecable
+      include Validatable
+
+      def initialize(attribute_name)
+        initialize_allow_blankable
+        initialize_contextable
+        initialize_messageable
+        initialize_rspecable
+        @attribute_name = attribute_name
+        @file_number_min = @file_number_max = nil
+      end
+
+      def description
+        "validate the limit files of :#{@attribute_name}"
+      end
+
+      def failure_message
+        "is expected to validate limit file of :#{@attribute_name}"
+      end
+
+      def file_number(number)
+        @file_number_min = @file_number_max = number
+        self
+      end
+
+      def file_number_min(number)
+        @file_number_min = number
+        self
+      end
+
+      def file_number_max(number)
+        @file_number_max = number
+        self
+      end
```

Comment:
These are the methods to be called upon the matcher instance. Therefore it should be `min` and `max` as mentioned in the `README.md` file

Reply:
and the instance variables `@min` and `@max`